Sprint-1/destructuring/exercise-3/exercise.js

```diff
@@ -6,3 +6,14 @@ let order = [
   { itemName: "Hot Coffee", quantity: 2, unitPricePence: 100 },
   { itemName: "Hash Brown", quantity: 4, unitPricePence: 40 },
 ];
+
+let totalCost = 0
+
+console.log("QTY      ITEM              TOTAL")
+
+order.forEach(({itemName, quantity, unitPricePence}) => {
+  const totalItemCost = (unitPricePence * quantity) / 100
+    totalCost += totalItemCost
+   
+    console.log(`${quantity}       ${itemName.padEnd(18)} ${totalItemCost.toFixed(2)}`)
+})
```

Comment:
It might be easier to align output using the string's built-in padding functions than using space characters.

---

The format of the output is not **exact**; something is omitted in your output and the spacing is off a bit.
Since the spec mentions `Pay attention to the exact formatting of the expected result.` It would be great if you can try to produce an output that matches the expected result.